js/stdlib/ts/loader.ts

```diff
@@ -8,10 +10,10 @@ export {getConnectorMode, getConnector};
 // XXX make an interface for Stdlib, return Promise<Stdlib>
 // The connectorMode arg is optional;
 // It will use REACH_CONNECTOR_MODE if 0 args.
-export async function loadStdlib(connectorModeOrEnv?: string | {[key: string]: string}): Promise<any> {
+export function loadStdlib(connectorModeOrEnv?: string | {[key: string]: string}): any {
```

Comment:
You're removing `async` because it isn't actually used? I think it might be good to leave it in because in the future another network might need some sort of initialization? Is that wrong to think? Maybe because we can't ensure that if you /DON'T/ use the loader that it will be "inited" first?

Reply:
Removing `async` is something I want to do anyway, because it lets you do `const stdlib = loadStdlib()` at the top-level instead of having to be inside of an `async` function to call it. My opinion is that any sort of "async" initialization should be hidden from the user via using promises internal to the stdlib's implementaiton.

Reply:
(Similar to how we made that change a while ago so that attach/deploy are no longer `async` functions, even though `deploy` does kick off an async deployment behind the scenes.)

Reply:
Sounds great